tests/components/ct-react/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "jsx": "react-jsx"
   },
   "include": [
-    "src"
+    "src",
```

Comment:
nit: we mix styling in tsconfig.json. Looks like other cases we put everything on one line, and others we do this.